src/block-components/typography/attributes.js

```diff
@@ -71,7 +71,7 @@ const typographyAttributes = {
 		default: false,
 	},
 	show: {
-		type: 'booleam',
+		type: 'boolean',
```

Comment:
Reverting this change because another PR is going to fix this.

```suggestion
		type: 'booleam',
```